project/MimaExcludes.scala

```diff
@@ -36,6 +36,9 @@ object MimaExcludes {
 
   // Exclude rules for 4.0.x
   lazy val v40excludes = v35excludes ++ Seq(
+    // [SPARK-44863][UI] Add a button to download thread dump as a txt in Spark UI
```

Comment:
@yaooqinn  would you mind move the two exclusions to `v35excludes`, so that we can also merge it to 3.5?

Reply:
OK, thanks, done